lib/utils/removeNullObjectValues.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'underscore';
```

Comment:
This is great moving it here. I swear I saw some other PR also adding a `utils.js` file, but I don't see it in `main`, so maybe it isn't merged yet... Would you mind seeing if you can find the other PR and then making sure that the utils are all being organized consistently?

Reply:
You're right, this was added in [this PR](https://github.com/Expensify/react-native-onyx/pull/329).

I also changed this in the [SQLite crash PR](https://github.com/Expensify/react-native-onyx/pull/334)